assets/schema_input.json

```
@@ -11,7 +11,19 @@
                 "type": "string",
                 "pattern": "^\\S+$",
                 "errorMessage": "Sample name must be provided and cannot contain spaces",
-                "meta": ["id"]
+                "meta": ["sample"]
+            },
+            "lane": {
+                "type": "integer",
+                "pattern": "^\\d+$",
```

```suggestion
                "type": "string",
                "pattern": "^L\\d+$",
```
Either that or the lane number should be changed in `assets/samplesheet.csv`

---

Good point. What do you use? L prefix, or just the number?

---

We decided to just go for integers since that's the format in the samplesheets we get from the sequencing instruments